profiler/src/ProfilerEngine/Datadog.Profiler.Native/OpSysTools.cpp

```diff
@@ -453,11 +449,12 @@ bool OpSysTools::IsSafeToStartProfiler(double coresThreshold, double& cpuLimit)
 
     auto sharedObjectPath = fs::path(info.dli_fname);
 
-    if (sharedObjectPath.filename() != wrapperLibraryName)
+    if (sharedObjectPath.filename() != wrapperLibraryName
+        && sharedObjectPath.filename() != wrapperLibraryNamePyroscope )
```

Comment:
this is the line that allows api wrapper with pyroscope name i.e.  `Pyroscope.Linux.ApiWrapper.x64.so`